Jint/Runtime/Modules/DefaultModuleLoader.cs

```diff
@@ -52,20 +140,20 @@ protected virtual ModuleLoaderResult LoadModule(Engine engine, Uri location)
         }
         catch (ParserException ex)
         {
-            ExceptionHelper.ThrowSyntaxError(engine.Realm, $"Error while loading module: error in module '{location}': {ex.Error}");
+            ExceptionHelper.ThrowSyntaxError(engine.Realm, $"Error while loading module: error in module '{resolved.Uri.LocalPath}': {ex.Error}");
             module = null;
         }
 
-        return new ModuleLoaderResult(module, location);
+        return module;
     }
 
-    protected virtual string LoadModuleSourceCode(Uri location)
+    protected virtual string ReadAllText(Uri uri)
```

Comment:
This is not reusable anymore in a sub-class. The fact that the checks are done outside will prevent custom implementation from loading urls.

Reply:
We'll need this one fixed before merging. I will do it otherwise.